mobile.css

```diff
@@ -0,0 +1,718 @@
+.msg:hover {
+  border: 2px solid #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
+}
+
+.submit-container {
+  display: flex;
+  justify-content: center;
+}
+
+.contact-submit {
+  display: flex;
+  font-family: inherit;
+  color: #edecff;
+  background-color: #7d73fc;
+  width: 100px;
+  height: 48px;
+  padding: 12px 40px 12px 35px;
+  font-size: 12px;
+  font-weight: 600;
+  align-items: center;
+  justify-self: center;
+}
+
+.contact-submit:active {
+  background-color: #7d73fc;
+}
+
+.contact-submit:hover {
+  background: #7d73fc;
+  box-shadow: 0 8px 16px rgba(125, 115, 252, 0.48);
+}
+
+.contact-submit:focus {
+  background: #5e42d7;
+}
+
+.contact-submit:disabled {
+  background: #2a2b33;
+}
+
+.footer {
+  display: flex;
+  flex-direction: column;
+  margin-top: 15px;
+  align-items: center;
+  color: #ebebff;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+@media screen and (min-device-width: 768px) {
+  body {
+    background-image:
+      url("/multimedia/circle-1-header.svg"),
+      url("/multimedia/circle-2-header.svg"),
+      url("/multimedia/circle-4.svg"),
+      url("/multimedia/circle-5.svg"),
+      url("/multimedia/circle-6.svg"),
+      url("/multimedia/circle-7.svg");
+    background-repeat:
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat,
+      no-repeat;
+    background-size:
+      300px 300px,
+      325px 325px,
+      330px 1000px,
+      280px 1200px,
+      600px 280px,
+      430px 1000px;
+    background-position:
+      top 198px left 69px,
+      left 1180px top 520px,
+      left -10px top 1150px,
+      right 0 top 1600px,
+      left -100px top 2500px,
+      right -10px top 2450px;
+  }
+
+  .main-page-header {
+    display: none;
+  }
+
+  .header-hidden {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #ffff;
+    padding: 0 142px;
+    margin-bottom: 0;
+  }
+
+  .nav-bar {
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav-links {
+    display: flex;
+    gap: 20px;
+    font-size: 0.93rem;
+  }
+
+  .message {
+    display: flex;
+    align-items: center;
+    padding-bottom: 4px;
+    align-content: center;
+  }
+
+  .work-element-hidden {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #ffff;
+    margin-bottom: 0;
+    padding: 0 40px;
+  }
+
+  .logo {
+    padding: 10px 0;
+  }
+
+  .open-menu {
+    padding: 10px 0;
+  }
+
+  .headline {
+    padding: 130px 142px 114px;
+    gap: 88px;
+  }
+
+  .name {
+    font-size: 5.62rem;
+    text-align: center;
+    line-height: 105px;
+  }
+
+  .occupation {
+    font-size: 2rem;
+    margin-bottom: 0;
+    padding: 0;
+    text-align: center;
+    line-height: 47px;
+  }
+
+  .about-text {
+    margin: 0;
+    padding: 0 10px 45px;
+    text-align: justify;
+    text-align-last: center;
+    line-height: 32px;
+    left: 0.5px;
+    width: 100%;
+  }
+
+  .about-text-links {
+    padding: 0 354px;
+  }
```

Comment:
- [ ] [OPTIONAL] It will be better to use relative measurements like `percent and vw` to make your website more responsive on all screens. [read more](https://www.mariokandut.com/absolute-and-relative-units-in-css/)

Reply:
@omarsalem7 I'll try that thank you! :raised_hands: